### PR TITLE
[W5.7][T12-4] FAN WENKAI

### PR DESCRIPTION
**Pull request by jasonvank**

### CODE REVIEW

File: test/java/seedu/addressbook/parser/ParserTest.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import seedu.addressbook.commands.DeleteCommand;
 import seedu.addressbook.commands.ExitCommand;
 import seedu.addressbook.commands.FindCommand;
+import seedu.addressbook.commands.FindPhoneNumberCommand;
```
Comment (zhengwei143) [inline test/java/seedu/addressbook/parser/ParserTest.java:21]: Might want to add in a few tests for the parser of the new command in this file

File: src/seedu/addressbook/data/person/Phone.java [hunk 2]
```diff
@@ -53,6 +53,11 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    public String getPhoneNumber() {
+        System.out.println("inside" + this.value);
```
Comment (eugenepeh) [inline src/seedu/addressbook/data/person/Phone.java:57]: Is this suppose to be a `debug code`?

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -36,11 +36,31 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // contains null
+        assertContainsNull(null, null);
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:42]: This is the last week LO and should not be inside this PR.
Do remember to create a clean branch every time you work on a new feature/modification.

**Review by eugenepeh — APPROVED** [review]
Good job on completing the activity.

Some things to note though, how would you justify the value of this feature compare to the `find` command? Also, do remember to update the test cases as well to ensure that your feature is working properly as intended.

Some comments added. Please close the PR after reading the comments.

**Pull request closed without merging**